TophatCtl/Commands/List/List+Apps.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  List+Apps.swift
+//  tophatctl
+//
+//  Created by Lukas Romsicki on 2024-12-04.
+//  Copyright © 2024 Shopify. All rights reserved.
+//
+
+import ArgumentParser
+import TophatControlServices
+
+extension List {
+	struct Apps: AsyncParsableCommand {
+		static let configuration = CommandConfiguration(
+			abstract: "Lists all apps configured for Quick Launch in Tophat."
+		)
+
+		func run() async throws {
+			checkIfHostAppIsRunning()
+
+			let reply = try await TophatRemoteControlService().send(request: ListAppsRequset())
+
+			print("Apps:")
+
+			for app in reply.apps {
+				print()
+
+				print("‣\u{001B}[1m", app.name, "\u{001B}[22m")
+				print("  Identifier:", app.id)
+				print("  Platforms:", app.platforms.map { String(describing: $0) }.joined(separator: ", "))
+				print("  Recipes:", app.recipeCount)
```

Comment:
It could be nice to see the recipes too, but this is already 👌 

```
➜  Debug ./tophatctl list apps
Apps:

‣ Shopify iOS Nightly
  Identifier: B4037010-D52E-4ED3-921A-811D62089AB5
  Platforms: iOS
  Recipes: 2
```